docker-compose.yml

```diff
@@ -106,5 +104,4 @@ services:
      OO_SERVICE_EMAIL: "openoversightchi@lucyparsonslabs.com"
      TIMEZONE: "America/Chicago"
    volumes:
-     - ./OpenOversight/:/usr/src/app/OpenOversight/:z
-     - ./build/test-results:/test-results
+     - ./:/usr/src/app:z
```

Comment:
As opposed to three different statements that pull in individual files/folders, this does it all more gracefully. From my understanding, it's also more idiomatic Docker styling.